package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-aria-menubutton",
-  "version": "4.3.1",
+  "version": "4.3.2",
```

Comment:
Hi David, finally found some time, yeah! Does this mean 4.4 or 5.0.0? Not sure if breaking changes mean a completely new version.